src/static/js/routes/common.js

```diff
@@ -4,40 +4,31 @@
 
 'use strict';
 
-var $ = require( 'jquery' );
+// Vendor libraries.
+require( 'jquery' );
+require( 'jquery-easing' );
+require( 'cf-expandables' );
 
-$( document ).ready( function() { // eslint-disable-line max-statements, no-inline-comments, max-len
+// Global modules.
+require( '../modules/beta-banner-state' ).init();
+require( '../modules/nav-primary.js' ).init();
+require( '../modules/secondary-nav-toggle' ).init();
+require( '../modules/mobile-primary-nav.js' ).init();
+require( '../modules/desktop-primary-nav.js' ).init();
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/322a52d2643c038cb1d8b0a5ecee4450ca9ea8da